examples/tools/cmake/cmake_toolchain/build_project_cmake_presets.rst

```diff
@@ -50,7 +50,7 @@ If you are using a multi-configuration generator:
 
 .. code:: bash
 
-    $ cmake --preset conan-default
+    $ cmake --preset conan-debug
```

Comment:
Multi-configuration generators such as ``Visual Studio`` or ``Ninja Multi-Config`` will generate ``conan-default`` configure profile. It is only ``conan-debug`` if it is not a multi-config generator.